File: problem-solving/beecrowd/1019-conversao-de-tempo/src/main.py
n = int(input())
hours = minutes = seconds = 0

if n >= 60:
    minutes = n // 60
    seconds = n % 60
else:
    seconds = n

if minutes >= 60:
    hours = minutes // 60
    minutes = minutes % 60

print(f"{hours}:{minutes}:{seconds}")

File: problem-solving/beecrowd/1019-conversao-de-tempo/src/main.c
#include <stdio.h>

int main() {
  unsigned n, hours=0, minutes=0, seconds;

  scanf("%u", &n);

  if (n >= 60) {
    minutes = n / 60;
    seconds = n % 60;
  }else{
    seconds = n;
  }
  if (minutes >= 60){
    hours = minutes / 60;
    minutes = minutes % 60;
  }

  printf("%u:%u:%u\n",hours, minutes, seconds);
}
